### PR TITLE
Cleaninggggggg

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/node/streams/piping-readable-streams.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/node/streams/piping-readable-streams.md?ref=7ba9a820d2b6b468c85bafcc0c6c1dbffe2e402f)

File: javascript/node/streams/piping-readable-streams.md [hunk 1]
```diff
@@ -1,26 +1,10 @@
...
-  javascript.node-standard-library-os.0: 20
-  javascript.events-asynchronous-operations.1: 20
-
-tags:
-  - introduction
-  - workout
-  - deep
-
 links:
   - '[Piping streams](https://www.sandersdenardi.com/readable-writable-transform-streams-node/){website}'
```
Comment (kapnobatai137) [inline javascript/node/streams/piping-readable-streams.md]:
```suggestion
  - '[Piping Streams](https://www.sandersdenardi.com/readable-writable-transform-streams-node/){website}'
```

File: javascript/node/streams/piping-readable-streams.md [hunk 2]
```diff
@@ -58,30 +44,30 @@ The `pipe()` function is used in the above example to write the content of `file
...
+* `pipe`
+* `w`
+* `fs.readStream`
+* `readStream`
+* `writeStream`
+* `fs.writeStream`
+* `transfer`
+* `move`
+* `fileA`
+* `fileB`
```
Comment (kapnobatai137) [inline javascript/node/streams/piping-readable-streams.md]: is there any benefit to enclosing these in backticks? both 
```
* `this`
* this
```
 will display the same in the app

Comment (nem035) [reply javascript/node/streams/piping-readable-streams.md]: Nope, I just did it cause I forgot if it renders properly in the app or not and was lazy to test it 😆

Comment (kapnobatai137) [reply javascript/node/streams/piping-readable-streams.md]: perfect then :smile:

**Pull request merged**